src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -26,9 +27,19 @@ const ExecuteTxButton = ({
   const txNonce = isMultisigExecutionInfo(txSummary.executionInfo) ? txSummary.executionInfo.nonce : undefined
   const isPending = useIsPending(txSummary.id)
   const { setSelectedTxId } = useContext(ReplaceTxHoverContext)
+  const safeSDK = getSafeSDK()
 
   const isNext = txNonce !== undefined && txNonce === safe.nonce
-  const isDisabled = !isNext || isPending
+  const isDisabled = !isNext || isPending || !safeSDK
+
+  const tooltipTitle =
+    isDisabled && !isNext
+      ? `Transaction ${safe.nonce} must be executed first`
+      : isPending
+      ? `There's a pending transaction`
```

Comment:
I like these new messages but would suggest to document them in the test plan of this PR. If possible, I would also suggest to extract this into a helper function that can be reused in all three buttons and write a few unit tests for it.

Reply:
Extracted the util and added the tests